src/commands/index.ts

```diff
@@ -25,4 +29,16 @@ export function registerCommands(context: ExtensionContext) {
   registerDirectoryCommands(context);
   registerTourCommands(context);
   registerScratchCommands(context);
+
+  vscode.commands.registerCommand("gistpad.runInlineEval", async (code) => {
+    const vm = new NodeVM({
+      require: {
+        external: true,
+        builtin: ["*"]
+      },
+      wrapper: "none"
+    });
+    let result = vm.run(code);
+    await vscode.window.showInformationMessage(result);
```

Comment:
Instead of displaying the results in an info message, should we consider writing it to an output pane or terminal window?  The main things I'm thinking about are: syntax highlighting, medium-to-large size JSON objects, etc. To make this experience really "shine", I wonder if info messages can truly shine.

Maybe we could explore hosting the React `console-feed` component in a web view and opening that to the side when you run a code block? This might even be useful for JavaScript-only playgrounds that can use the console window instead of a standard webview :D